<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractiveConsoleWidget</class>
 <widget class="QWidget" name="InteractiveConsoleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interactive Console</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="outputPlainTextEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="promptLabel">
       <property name="text">
        <string>&gt;&gt;&gt; </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputLineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
